component-runtime-manager/src/test/java/org/talend/sdk/component/runtime/manager/ComponentManagerTest.java

```diff
@@ -76,6 +78,10 @@
 
     private final PluginGenerator pluginGenerator = new PluginGenerator();
 
+    public static final String M2_PROP = "talend.component.manager.m2.repository";
+
+    public static final String SETTINGS_PROP = "talend.component.manager.m2.settings";
```

Comment:
Didn't know that we have this one.
Also can't find it in documentation

Reply:
yea, you're right it's undocumented... (will fix that later)